Kickstarter-iOS/Views/Cells/DiscoveryPostcardCell.swift

```diff
@@ -348,4 +349,12 @@ internal final class DiscoveryPostcardCell: UITableViewCell, ValueCell {
   @objc fileprivate func saveButtonTapped(_ button: UIButton) {
     self.watchProjectViewModel.inputs.saveButtonTapped(selected: button.isSelected)
   }
+
+  func roundImage(_ url: URL) -> UIImage? {
```

Comment:
`private`?

Reply:
Can we also use more descriptive signature for this function?

I.e. `roundImage(for url: URL) { .. }`

Reply:
Yup going to correct this too